README.md

```diff
@@ -32,5 +32,30 @@ Okay, I don't understand how to add preview picture, I will try to google it aga
 
 And here is my pic from rviz:
 ![picture](https://github.com/UralmashFox/not_Skin_bastard/blob/master/Thespot.png)
+
+Some testing fun! Let's try to test my spot model. Open your terminal and do all that things upper. That will make us free from running roscore and pushing instructions when integrational test will be.
```

Comment:
AAAAA, remove build folder and all useless files. How to check 427 files, are you kidding?

Reply:
done

Reply:
i see that you have removed but 182 changed files :(

Reply:
there are 12 files in the average, :D